tests/unit/accounts/test_views.py

```diff
@@ -202,7 +202,7 @@ def test_post_validate_redirects(self, monkeypatch, pyramid_request,
         ("expected_next_url, observed_next_url"),
         [
             ("/security/", "/security/"),
-            ("http://example.com", "/"),
+            ("http://example.com", "/manage/projects"),
```

Comment:
@di Do I keep this "/manage/projects" as it is or shall I change it? I see this test failing for some reason.

Reply:
The tests are failing because it's trying to call `route_path`, you'll need to mock it out for this test as well, and this value will become something like `'/the-redirect'` (or whatever you set the mocked path to be).

Reply:
@di Just a quick question :

If I have 

`assert pyramid_request.route_path.calls == [pretend.call('manage.projects')]` wouldn't it fail the test case with the parameter `/securiry` ?

Reply:
We don't necessarily need to assert that `route_path` is called here, I think it'd be fine to omit that assertion.